app/views/case_workers/admin/allocations/_re_allocation.html.haml

```diff
@@ -1,16 +1,16 @@
 = content_for :v2_stylesheets do
-  = stylesheet_link_tag "v2/application", media: "all"
+  = stylesheet_link_tag 'v2/application', media: 'all'
 
-= render partial: 'errors/error_summary', locals:{ ep: @allocation, error_summary: t('.error_summary')}
+= render partial: 'errors/error_summary', locals: { ep: @allocation, error_summary: t('.error_summary') }
 
 .grid-row
   .column-full
-    = render partial: 'layouts/header', locals: {page_heading: t('.page_heading')}
+    = render partial: 'layouts/header', locals: { page_heading: t('.page_heading') }
 
 .grid-row
   .column-full
     = form_tag case_workers_admin_allocations_path, method: :get do
```

Comment:
**NOT a blocker** but could try to sort out the nested form issue here as mentioned here [../allocation.html.haml](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/2994/files#r328094880) and use `form_with` too

Reply:
As discussed, this instance is not nested

Reply:
👍